manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -56,16 +56,6 @@ postInstall:
       pullPolicy: IfNotPresent
       pullSecrets: []
     extraNamespaces: []
-  probeWebhook:
```

Comment:
why was this removed?

Reply:
was not intentional, I will readd it asap